### ISSUE TITLE
[BUG] [Linter]

### ISSUE THREAD

**Issue by schinckel** [djlint/djLint#831]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: macOS
 - Python Version: 3.11
 - djLint Version: 1.34.0
 - template language: DTL

## Issue
maps:// urls are incorrectly detected as internal urls.

## How To Reproduce

HTML file contains:

`<a href="maps://?ll={{ obj.address.position.coords.1 }},{{ obj.address.position.coords.0 }}&z=20">`

## Contents of .djlintrc/pyproject.toml [tool.djlint]

```
[tool.djlint]
blank_line_after_tag = "load,extends"
indent = 2
custom_blocks = "nested_widget,capture,flag,switch"
format_attribute_template_tags = true
max_attribute_length = 1
line_break_after_multiline_tag = true
max_blank_lines = 3
ignore="H021,T002"
```
